tests/system_tests_tcp_adaptor.py

```diff
@@ -556,18 +648,22 @@ def router(name, mode, connection, extra=None):
                             time.sleep(0.25)
                             break
 
-        cls.logger.log("TCP_TEST All tcpListeners are active")
+        cls.logger.log("**************TCP_TEST All tcpListeners are active**************")
```

Comment:
This makes using Ctrl+F for `***` to find failed tests less effective. One more false match :p